magenta/interfaces/midi/midi_hub.py

```diff
@@ -177,22 +192,40 @@ def run(self):
         self._start_time,
         now + period - ((now - self._start_time) % period))
     while self._stop_time is None or self._stop_time > next_tick_time:
+      self._count += 1
```

Comment:
Lets call this 'metric_position' (or something similar) and do the mod on increment.
That will make its use a bit more clear.